ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsTransactionSpec.scala

```diff
@@ -82,6 +84,31 @@ class KVUtilsTransactionSpec extends AnyWordSpec with Matchers {
     val p0 = mkParticipantId(0)
     val p1 = mkParticipantId(1)
 
+    "add the key of an exercised contract as a submission input" in KVTest.runTestWithSimplePackage(
```

Comment:
I take it this test fails without the changes to `InputsAndEffects`?

Reply:
Yes.